plugin/storage/cassandra/spanstore/dbmodel/tag_filter_drop_all.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2019 The Jaeger Authors.
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dbmodel
+
+import (
+	"github.com/jaegertracing/jaeger/model"
+)
+
+// TagFilterDropAll filters all fields of a given type.
+type TagFilterDropAll struct {
+	dropTags        bool
+	dropProcessTags bool
+	dropLogs        bool
+}
+
+// NewTagFilterDropAll return a filter that filters all of the specified type
+func NewTagFilterDropAll(dropTags bool, dropProcessTags bool, dropLogs bool) TagFilterDropAll {
```

Comment:
I would return a pointer here and change receivers accordingly. In practice this will be used by callers that will assign the result to something of type TagFilter, which is an interface, meaning that this struct would still escape to the heap in order to be converted to the interface.